src/_pytest/runner.py

```diff
@@ -76,11 +85,11 @@ def pytest_terminal_summary(terminalreporter: "TerminalReporter") -> None:
         dlist = dlist[:durations]
 
     for i, rep in enumerate(dlist):
-        if verbose < 2 and rep.duration < 0.005:
+        if verbose < 2 and rep.duration < durations_min:
             tr.write_line("")
             tr.write_line(
-                "(%s durations < 0.005s hidden.  Use -vv to show these durations.)"
-                % (len(dlist) - i)
+                "(%s durations < %gs hidden.  Use -vv to show these durations.)"
```

Comment:
Is "%gs" valid?

Reply:
```suggestion
                "(%s durations < %ss hidden.  Use -vv to show these durations.)"
```
Typo?

Reply:
`%gs` is valid and used there on purpose, in order to print variable precision, aka human numbers. "1.000" will render as "1" but "1.010" as "1.01".